csv.inc.php

```diff
@@ -295,6 +284,5 @@ private function printToScreen($row) //See source 5
     5:http://php.net/manual/en/function.mysql-field-name.php: Print row help
     6:http://php.net/manual/en/function.str-pad.php: Ref 1 to padding
     7:https://www.w3schools.com/php/func_string_str_pad.asp: Ref 2 to padding
-
  *
- */
+ */
```

Comment:
This is a rule many aren't common with, but you should have a newline at the end of your scripts.